### PR TITLE
Install mongo 2.6

### PR DESCRIPTION
**Pull request by nathanleiby**
We also need support for installing Mongo 2.6.

Between 2.4 and 2.6, the apt packages used by Mongo changed, so it looks a bit different.

@mohit should I combine this now with Mongo install 2.4, or keep both?

Docs re: installing Mongo on ubuntu: 

- guide: https://futurestud.io/tutorials/how-to-install-mongodb-2-6-on-ubuntudebian
- official docs: https://docs.mongodb.com/manual/tutorial/install-mongodb-on-ubuntu/

### CODE REVIEW

**Comment by mohit** [comment]
I don't have a strong opinion either way. I'm not too worried about drift in these scripts so it's fine.

File: circleci/mongo-install-2.6 [hunk 1]
```diff
@@ -0,0 +1,31 @@
...
+sudo apt-get update
+sudo apt-get install \
+    mongodb-org=$VERSION \
+    mongodb-org-server=$VERSION \
+    mongodb-org-shell=$VERSION \
+    mongodb-org-mongos=$VERSION \
+    mongodb-org-tools=$VERSION
+
+echo "Checking Mongo version..."
+mongo --version
```
Comment (mohit) [inline circleci/mongo-install-2.6]: do we need to restart like for 2.4

Comment (nathanleiby) [inline circleci/mongo-install-2.6]: I looked in the logs and actually the restart is unnecessary for 2.4, too. I'll remove it there.

**Comment by nathanleiby** [comment]
@mohit I refactored to allow installing both from same script. The only different code is that apt packages to install, which is nice!

**Pull request merged**